src/config/index.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   },
   INFURA_API_KEY: process.env.INFURA_API_KEY || "bb46da3f80e040e8ab73c0a9ff365d18",
   INFURA_PROJECT_ID: "1f1ff2b3fca04f8d99f67d465c59e4ef",
-  ALCHEMY_API_KEY: process.env.ALCHEMY_API_KEY || "pV9JmoYcDFFBihXlJWt6FSDxzNAVWxdT",
+  ALCHEMY_API_KEY: process.env.ALCHEMY_API_KEY || "483Uqp_0SP0NnXI0secbJgrSRXv4OXaj",
```

Comment:
Just curious only, is any part of our frontend now uses the Alchemy API key instead of the Infura one?

Reply:
none at all. i can only assume it was to help those  `**.integration.test.ts` tests be more stable

Reply:
any concerns?

Reply:
No strong opinions about it, though, I’d be curious if there is a reason for using Alchemy for integration testing only but Infura for the actual functionalities?